Source/WebService/Blackjack.WebService/ClientApp/src/app/components/play-game-page/play-game-page.component.ts

```diff
@@ -0,0 +1,47 @@
+import { Component, OnInit } from "@angular/core";
+import { UserApi } from "../../../shared/api/user.api";
+import { UserDto } from "../../../shared/dtos/user.dto";
+
+@Component({
+  templateUrl: './play-game-page.component.html',
```

Comment:
Components already represent UI pages inherently. Naming these files play-game-page.component.xxxx is kind of redundant.

This is just me nitpicking but I personally would rename these files play-game.component.xxxx or game.component.xxxx